### PR TITLE
[W5][W09-1] Lim Zheng Jie Benjamin

### PR DESCRIPTION
**Pull request by LimZhengJieBenjamin**
Added a sort function to addressbook-level2.

### CODE REVIEW

**Comment by NgWeiXin** [comment]
I've seen the sort command and realised that the user has to call for Sort to Sort the List of person, might be better if the user doesn't need to type Sort command or the user can Sort based on Name/Address/Tags! 👍

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -151,6 +151,10 @@ Views all details of the 2nd person in the address book.
 `viewall 1` +
 Views all details of the 1st person in the results of the `find` command.
 
+== Clearing all entries : `sort`
```
Comment (okkhoy) [inline docs/UserGuide.adoc:154]: Ouch! Copy paste error?

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.commands;
+
+public class SortCommand extends Command {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/SortCommand.java:3]: Classes and their non-trivial methods need header comments

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:14]: Don't import everything using `*`. Import only th necessary classes

**Review by okkhoy — COMMENTED** [review]
Do remember to update the tests for each new feature implemented.
Please be careful in updating the documents.

**Comment by okkhoy** [comment]
@LimZhengJieBenjamin 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**